src/scene/skybox/sky.js

```diff
@@ -59,6 +59,15 @@ class Sky {
         this.projectedSkydomeCenterId = this.device.scope.resolve('projectedSkydomeCenter');
     }
 
+
+    applySettings(render) {
+        this.node.setLocalPosition(new Vec3(render.skyPosition ?? [0, 0, 0]));
+        this.node.setLocalScale(new Vec3(render.skyScale ?? [1, 1, 1]));
```

Comment:
and for completeness, expose 'rotation' as well please - this is handy for box projection if the room geometry is not aligned with the axes.
maybe those 3 that we set on the node should be `skyMeshPosition` `skyMeshRotation` and `skyMeshScale`, to not confuse them with the existing `skyboxRotation`, that rotates the texture

Reply:
I suggested in the other comment to do `skyTextureRotation` for the existing rotation prop but since you said it would change the existing API this seems more logical